day2/1/my_try_no_regex.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from datetime import datetime
+
+   
+def get_errors(path,err_code):
+    
+    log=[]
+    
+       
+    try:
+        with open(path) as f:
+            
+            
+            for line in f.readlines():
+                err=0
+                cnt=0
+                cnt2=0
+                for element in line.split():
```

Comment:
`for i, element in enumerate(line.split())`:

Reply:
"i" non mi è servito